sidebar-demo-swt/build.gradle

```diff
@@ -31,5 +31,5 @@ private static String getOsgiPlatformValue() {
         return 'cocoa.macosx.x86_64'
     }
 
-    throw new GradleException('unknown OS name: ' + osName)
+    throw new IllegalStateException('unknown OS name: ' + osName)
```

Comment:
Was this change included to make the exception that is thrown more specific?

Reply:
Yes, I was too lazy to create a separate PR just for this.
The idea is to not depend on Gradle-specific classes.